### PR TITLE
#258 Script to (re)define nrt references automatically

### PR DESCRIPTION
**Pull request by rosiereflo**

### CODE REVIEW

File: util/nrt_diff/defineTestReference.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/python
```
[the remark this answers is not in the document]
Comment (rosiereflo) [reply util/nrt_diff/defineTestReference.py]: I agree, but it is out of the scope of this commit IMHO

File: util/envDynawo.sh [hunk 2]
```diff
@@ -1359,6 +1360,10 @@ nrt_diff() {
   python $DYNAWO_NRT_DIFF_DIR/nrtDiff.py $1 $2 $3
 }
 
+nrt_ref() {
+  python $DYNAWO_NRT_DIFF_DIR/defineTestReference.py $1 $2 $3 $4 $5
```
[the remark this answers is not in the document]
Comment (rosiereflo) [reply util/envDynawo.sh]: For homogeneity I kept the format above. Will change both.

File: util/nrt_diff/defineTestReference.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+# simulation tool for power systems.
+
+import os
+import fnmatch
```
[the remark this answers is not in the document]
Comment (rosiereflo) [reply util/nrt_diff/defineTestReference.py]: No need to install anything to use it

File: util/envDynawo.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (gautierbureau) [reply util/envDynawo.sh]: ok

File: util/nrt_diff/defineTestReference.py [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    import lxml.etree
+except:
+    print("Error when trying to import lxml.etree")
+    sys.exit(1)
+from shutil import copyfile
+from optparse import OptionParser
+
+from nrtDiff import TestCase, REFERENCE_DATA_DIRECTORY_NAME, namespaceDYN, CompareTwoFiles, LogsSeparator, IDENTICAL
+
+branch_name = os.environ["DYNAWO_BRANCH_NAME"]
```
[the remark this answers is not in the document]
Comment (rosiereflo) [reply util/nrt_diff/defineTestReference.py]: Solved

**Pull request merged**